src/argilla/client/feedback/utils.py

```diff
@@ -110,7 +110,7 @@ def generate_pydantic_schema_for_metadata(
     metadata_fields, metadata_validators = {}, {}
 
     for metadata_property in metadata_properties:
-        if metadata_property.type not in MetadataPropertyTypes:
+        if metadata_property.type not in [arg.value for arg in MetadataPropertyTypes]:
```

Comment:
this can do the same, right?
```suggestion
        MetadataPropertyTypes(metadata_property.type)
```

Reply:
No it's not, accessing it as `MetadataPropertyTypes(metadata_property.type)` will fail if `metadata_property.type.value` is not in `[arg.value for arg in MetadataPropertyTypes`